inspirehep/modules/orcid/api.py

```diff
@@ -101,6 +101,7 @@ def push_record_with_orcid(recid, orcid, oauth_token, put_code=None, old_hash=No
                 data=orcid_xml,
                 put_code=put_code,
                 content_type='application/orcid+xml',
+                timeout=30,
```

Comment:
@turtle321 this is the value that we agreed on, right?

Reply:
It looks reasonable.
I am also fine with 60 secs for non "user facing" tasks.